tests/xfstests/boot.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Wait for system to boot
+# Maintainer: Nathan Zhao <jtzhao@suse.com>
+package boot;
+
+use strict;
+use 5.018;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->wait_boot;
```

Comment:
have you seen boot/boot_to_desktop.pm ? it's doing the same just with different timeout ( in your case 100 seconds timeout and 80 seconds in case of boot_to_desktop ). 20 seconds shouldn't be a problem and even if it is I would vote for modify boot_to_descktop instead of introducing yat-another-module for boot.

Reply:
I agree. I'll use boot/boot_to_desktop.pm instead.